content/blog/2022-03-31-training-and-saving-models-on-a-dedicated-runner.md

```diff
@@ -0,0 +1,284 @@
+---
+title:
+  Training and saving models with CML on a dedicated AWS EC2 runner (part 1)
+date: 2022-03-31
+description:
+  In this guide we will show how you can use CML to automatically retrain a
+  model and save its outputs to your Github repository using a provisioned AWS
+  EC2 runner.
+descriptionLong: |
+  We can use CML to automatically retrain models whenever data, model code,
+  or parameters change. In this guide we show how to create a pipeline that
+  provisions an AWS EC2 instance to retrain a model and save the output on
+  a regular basis. This way we can prevent drift by ensuring that our model
+  always uses the latest input data.
+picture: 2015-05-01/post-image.jpeg
+# pictureComment: Some _Comment_ (supports _basic_ [Markdown](link))
+author: rob_dewit
+# commentsUrl: https://discuss.dvc.org/t/february-22-community-gems/1078
+tags:
+  - CML
+  - Git
+  - Pipelines
+  - Dedicated runners
```

Comment:
Asked question in Slack if a more technical SEO grabbing term than dedicated runners.  May be fine as is.

Reply:
```suggestion
  - self-hosted runners
  - CI/CD
  - cloud compute
```